assignments/arrays.js

```diff
@@ -75,30 +75,51 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+/*console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);*/
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`${inventory[inventory.length-1].car_make} ${inventory[inventory.length-1].car_model}`);
```

Comment:
```suggestion
console.log(`${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);
```

Here they were trying to suggest that you pull the index value of the last car out into a separate variable, then reference that in your console log. The functionality is the exact same as what you had, but this way creates a variable that can then be used elsewhere (other functions, logs, etc.). This also makes code more readable.